frontend/src/drivers/socket.io.js

```diff
@@ -0,0 +1,56 @@
+export default {createSocketIODriver: createFakeSocketIODriver};
+
+import Rx from 'rx';
+import io from 'socket.io-client';
```

Comment:
`io` is just function importet from `socket.io-client` library. sometimes libraries can expose many separate functions.

`Rx` is object - general, big, library object with many, many methods inside.

I'm not sure if we should treat them in same way.